pom.xml

```diff
@@ -21,6 +21,7 @@
         <cql-engine.version>1.5.1</cql-engine.version>
         <spring.version>5.3.6</spring.version>
         <dagger.version>2.34.1</dagger.version>
+        <core.version>5.3.9</core.version>
```

Comment:
This is several versions behind the current fhir core version. Let's start out on the latest version.